item.rb

```diff
@@ -0,0 +1,22 @@
+require 'date'
+
+class Item
+  attr_reader :id, :archived, :publish_date
+  attr_accessor :label, :author, :genre
+
+  def initialize(publish_date, archived: false)
+    @id = Random.rand(1..1000)
+    @publish_date = publish_date
+    @archived = archived
+    @label = nil
+    @genre = nil
+  end
+
+  def can_be_archived?
+    Time.now.year - Date.parse(@publish_date).year > 10
+  end
+
+  def move_to_archive
+    @archived = can_be_archived?
+  end
+end
```

Comment:
- [ ]  Item class should have one Genre, Author, Label as you work in a group of three 😸 , the one to many assossiation should be present between  `item`  and `author` ,   `item`  and `genre`  and  `item`  and `author` ,  👍 as in the following example
```bash
class Item
  attr_reader :author

  ...

  def author=(author)
      @author = author
      author.items.push(self) unless author.items.include?(self)
  end

  ...
end
```